apis/ec2/v1alpha1/custom_types.go

```diff
@@ -60,28 +60,103 @@ type CustomVolumeParameters struct {
 // CustomVPCPeeringConnectionParameters are custom parameters for VPCPeeringConnection
 type CustomVPCPeeringConnectionParameters struct {
 	// The ID of the requester VPC. You must specify this parameter in the request.
-	VPCID *string `json:"vpcID,omitempty"`
+	// +crossplane:generate:reference:type=github.com/crossplane/provider-aws/apis/ec2/v1beta1.VPC
+	// +crossplane:generate:reference:refFieldName=VPCIDRef
+	// +crossplane:generate:reference:selectorFieldName=VPCIDSelector
+	VPCID *string `json:"vpcId,omitempty"`
 	// VPCIDRef is a reference to an API used to set
 	// the VPCID.
 	// +optional
-	VPCIDRef *xpv1.Reference `json:"vpcIDRef,omitempty"`
+	VPCIDRef *xpv1.Reference `json:"vpcIdRef,omitempty"`
```

Comment:
These are breaking changes so we'd need to bump `VPCPeeringConnection` to `v1alpha2` or `v1beta1`